web/backend/config.json

```diff
@@ -0,0 +1,15 @@
+{
+<<<<<<< Updated upstream
+  "FRONT_END_URL": "https://localhost:3000",
+  "DELA_NODE_URL": "localhost:12345",
+  "SESSION_SECRET": "session secret",
+  "PUBLIC_KEY": "b0e87b48610cd96c3e78ac7b8a217f39a3479941f6728425bad16d62b0c02be5",
+  "PRIVATE_KEY": "983f95ee311e0f545b8bc72f49c0f9709046ff9cdca4cf77141d163b6c43c60f"
+=======
+  "FRONT_END_URL" : "https://localhost:3000",
+  "DELA_NODE_URL" : "localhost:12345",
+  "SESSION_SECRET" : "session secret",
+  "PUBLIC_KEY" : "b0e87b48610cd96c3e78ac7b8a217f39a3479941f6728425bad16d62b0c02be5",
+  "PRIVATE_KEY" : "983f95ee311e0f545b8bc72f49c0f9709046ff9cdca4cf77141d163b6c43c60f"
+>>>>>>> Stashed changes
```

Comment:
Sorry I think I had to stash it to checkout your branch and it didn't merge correctly this file...
@nkcr do we need this file, it has been untracked for some weeks now ?